src/main/java/org/myrobotlab/framework/interfaces/JsonInvoker.java

```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.framework.interfaces;
+
+import org.myrobotlab.framework.Message;
+
+public interface JsonInvoker {
+
+  /**
+   * No parameter method
+   * @param method
+   * @return
+   */
+  public Object invoke(String method);
+
+  /**
+   * Encoded parameters as a JSON String (encoded once!)
+   * @param method
+   * @param encodedParameters
+   * @return
+   */
+  public Object invoke(String method, String encodedParameters);
```

Comment:
seems like if we are going to support json invoking, we should have default implementation in the interface to handle it.

Reply:
ya .. probably possible